distro/exporter.go

```diff
@@ -236,6 +238,51 @@ func newOTLPMetricsExporter(c *exporterConfig) (metric.Exporter, error) {
 	return otlpmetricgrpc.New(ctx, opts...)
 }
 
+type logsExporterFunc func(*exporterConfig) (log.Exporter, error)
+
+// logsExporters maps environment variable values to logs exporter creation
+// functions.
+var logsExporters = map[string]logsExporterFunc{
+	// OTLP gRPC exporter.
+	"otlp": newOTLPlogExporter,
+	// None, explicitly do not set an exporter.
+	"none": nil,
+}
+
+func logsExporter(log logr.Logger) logsExporterFunc {
+	key := envOr(otelLogsExporterKey, defaultLogsExporter)
+	lef, ok := logsExporters[key]
+	if !ok {
+		err := fmt.Errorf("invalid %s: %q", otelLogsExporterKey, key)
+		log.Error(err, "using default %s: %q", otelLogsExporterKey, defaultLogsExporter)
+
+		return logsExporters[defaultLogsExporter]
+	}
+	return lef
+}
+
+func newOTLPlogExporter(c *exporterConfig) (log.Exporter, error) {
+	ctx := context.Background()
+
+	var opts []otlploggrpc.Option
```

Comment:
What about HTTP support?

Reply:
This is OK.
It is only needed for our trace and metrics ingest endpoint. We do not have logs ingest (sic!).

Reply:
What about sending logs to a local collector over HTTP?

Reply:
@MrAlias 

We are always sending to collector using gRPC for all signals.
See:
- https://github.com/signalfx/splunk-otel-go/blob/46aa0ab7055386411c699579cbf38dd88f345bf3/distro/exporter.go#L60-L92
- https://github.com/signalfx/splunk-otel-go/blob/46aa0ab7055386411c699579cbf38dd88f345bf3/distro/exporter.go#L205-L237

There was never an ask to add support for `OTEL_EXPORTER_OTLP_PROTOCOL`.

Maybe we can consider reusing [`autoexport` ](https://pkg.go.dev/go.opentelemetry.io/contrib/exporters/autoexport) module as a separate PR. What do you think?

Reply:
I'm confused. Why do we support HTTP for the other signals?

Reply:
Because  ingest  is (or was) not supporting gRPC.

```
// Direct ingest to Splunk Observabilty Cloud using HTTP/protobuf. 
```

Reply:
> Even if we do support gRPC as well I heard that it is better to use `http/protobuf` with our backend. This is why AFAIK all our distros are using `http/protobuf` when `SPLUNK_REALM` is used. We use `grpc` in other cases for backwards compatibility.

Right ... so shouldn't we support HTTP? This is only adding gRPC exporters but you are saying that HTTP is preferred, right?

Reply:
> Right ... so shouldn't we support HTTP? This is only adding gRPC exporters but you are saying that HTTP is preferred, right?

It will break users that use `OTEL_EXPORTER_OTLP_ENDPOINT` which targets gRPC endpoints. 

Reference: https://docs.splunk.com/observability/en/gdi/get-data-in/application/go/configuration/advanced-go-otel-configuration.html